### ISSUE TITLE
Throws errors when used in same directory with Cython build in place

### ISSUE THREAD

**Issue by KyleKing** [autocomplete-python/autocomplete-python#224]
[Enter steps to reproduce below:]
1. Run `python setup.py build_ext --inplace`
2. Return to the python file and try to autocomplete a variable name, which throws a long list of errors [Note: doesn't effect autocomplete on `print` and other built-in functions]

-> This occurs only when the cython executable `*.so` file is present. Deleting that file and restarting Atom clears up the problem. How do I tell autocomplete-python to ignore this file?

[Troubleshooting Steps]
- I added any *.o or *.so to the global Atom file ignore glob pattern:

<img width="801" alt="settings -users-kyleking-developer-werk-7_probearmcamera_scripts atom today at 8 17 24 am" src="https://cloud.githubusercontent.com/assets/3784339/17553324/47d12cb4-5ed3-11e6-9934-c256a982003b.png">

<img width="465" alt="untitled -users-kyleking-developer-werk-7_probearmcamera_scripts atom today at 8 03 03 am" src="https://cloud.githubusercontent.com/assets/3784339/17552914/e34cf108-5ed0-11e6-837b-9bc80d175023.png">

**Atom Version**: 1.9.7
**System**: Mac OS X 10.11.6
**Thrown From**: [autocomplete-python](https://github.com/sadovnychyi/autocomplete-python) package, v1.8.6
### Stack Trace

Uncaught SyntaxError: Unexpected token /

```
At file:///Applications/Atom.app/Contents/Resources/app.asar/static/index.html#%7B%22initialPaths%22%3A%5B%22%2FUsers%2Fkyleking%2FDeveloper%2FWerk%2F7_ProbeArmCamera_Scripts%22%5D%2C%22locationsToOpen%22%3A%5B%7B%22pathToOpen%22%3A%22%2FUsers%2Fkyleking%2FDeveloper%2FWerk%2F7_ProbeArmCamera_Scripts%22%2C%22initialLine%22%3Anull%2C%22initialColumn%22%3Anull%2C%22forceAddToWindow%22%3Afalse%7D%5D%2C%22windowInitializationScript%22%3A%22%2FApplications%2FAtom.app%2FContents%2FResources%2Fapp.asar%2Fsrc%2Finitialize-application-window.js%22%2C%22resourcePath%22%3A%22%2FApplications%2FAtom.app%2FContents%2FResources%2Fapp.asar%22%2C%22devMode%22%3Afalse%2C%22safeMode%22%3Afalse%2C%22profileStartup%22%3Afalse%2C%22clearWindowState%22%3Afalse%2C%22env%22%3A%7B%22TMPDIR%22%3A%22%2Fvar%2Ffolders%2F1f%2Fgd24l7210d3d8crp0clcm4440000gn%2FT%2F%22%2C%22__CF_USER_TEXT_ENCODING%22%3A%220x1F5%3A0x0%3A0x0%22%2C%22HOME%22%3A%22%2FUsers%2Fkyleking%22%2C%22SHELL%22%3A%22%2Fbin%2Fbash%22%2C%22Apple_PubSub_Socket_Render%22%3A%22%2Fprivate%2Ftmp%2Fcom.apple.launchd.d1HGstNvTW%2FRender%22%2C%22SSH_AUTH_SOCK%22%3A%22%2Fprivate%2Ftmp%2Fcom.apple.launchd.D2S95DB8rr%2FListeners%22%2C%22PATH%22%3A%22%2Fusr%2Fbin%3A%2Fbin%3A%2Fusr%2Fsbin%3A%2Fsbin%22%2C%22LOGNAME%22%3A%22kyleking%22%2C%22XPC_SERVICE_NAME%22%3A%22com.github.atom.320992%22%2C%22COMMAND_MODE%22%3A%22unix2003%22%2C%22USER%22%3A%22kyleking%22%2C%22SECURITYSESSIONID%22%3A%22186a7%22%2C%22XPC_FLAGS%22%3A%220x0%22%2C%22GOOGLE_API_KEY%22%3A%22AIzaSyAQfxPJiounkhOjODEO5ZieffeBv6yft2Q%22%2C%22ATOM_HOME%22%3A%22%2FUsers%2Fkyleking%2F.atom%22%7D%2C%22appVersion%22%3A%221.9.7%22%2C%22atomHome%22%3A%22%2FUsers%2Fkyleking%2F.atom%22%2C%22shellLoadTime%22%3A416%7D:1

SyntaxError: Unexpected token /
    at Object.parse (native)
    at module.exports._deserialize (/Users/kyleking/.atom/packages/autocomplete-python/lib/provider.coffee:328:23)
    at provider.BufferedProcess.stdout (/Users/kyleking/.atom/packages/autocomplete-python/lib/provider.coffee:51:10)
    at Socket.<anonymous> (/Applications/Atom.app/Contents/Resources/app.asar/src/buffered-process.js:87:13)
    at emitOne (events.js:90:13)
    at Socket.emit (events.js:182:7)
    at readableAddChunk (_stream_readable.js:153:18)
    at Socket.Readable.push (_stream_readable.js:111:10)
    at Pipe.onread (net.js:531:20)
```
### Commands

```
     -0:02.4.0 editor:newline (atom-text-editor.editor.is-focused)
     -0:01.1.0 autocomplete-plus:confirm (atom-text-editor.editor.is-focused.autocomplete-active)
     -0:00.3.0 autocomplete-plus:activate (atom-text-editor.editor.is-focused)
```
### Config

``` json
{
  "core": {
    "ignoredNames": [
      ".git",
      ".hg",
      ".svn",
      ".DS_Store",
      "._*",
      "Thumbs.db",
      "*.pyc",
      ".DS_Store",
      "*.zip",
      "*.xls*",
      "*.o",
      "*.so"
    ],
    "projectHome": "~/Developer/My-Programming-Sketchbook",
    "themes": [
      "atom-material-ui",
      "atom-material-syntax"
    ]
  }
}
```
### Installed Packages

``` coffee
# User
atom-beautify, v0.29.10 (inactive)
atom-material-syntax, v0.4.6 (active)
atom-material-ui, v1.3.3 (active)
atomic-chrome, v0.3.0 (active)
autocomplete-python, v1.8.6 (active)
file-icons, v1.7.18 (active)
git-time-machine, v1.5.3 (inactive)
highlight-selected, v0.11.2 (active)
language-batchfile, v0.4.0 (active)
language-cython, v0.3.0 (active)
language-gitignore, v0.2.0 (active)
linter, v1.11.16 (active)
linter-annotations, v1.0.1 (active)
linter-flake8, v1.13.2 (active)
minimap, v4.24.7 (active)
minimap-cursorline, v0.2.0 (active)
minimap-find-and-replace, v4.5.1 (active)
oceanic-next, v0.1.2 (inactive)
open-terminal-here, v2.1.4 (inactive)
project-manager, v2.9.7 (active)
python-tools, v0.6.8 (active)
script, v3.8.3 (inactive)
set-syntax, v0.3.0 (active)
atom-dark-syntax, v0.27.0 (inactive)
atom-dark-ui, v0.51.0 (inactive)
atom-light-syntax, v0.28.0 (inactive)
atom-light-ui, v0.43.0 (inactive)
base16-tomorrow-dark-theme, v1.1.0 (inactive)
base16-tomorrow-light-theme, v1.1.1 (inactive)
one-dark-ui, v1.3.2 (inactive)
one-light-ui, v1.3.2 (inactive)
one-dark-syntax, v1.2.0 (inactive)
one-light-syntax, v1.2.0 (inactive)
solarized-dark-syntax, v1.0.2 (inactive)
solarized-light-syntax, v1.0.2 (inactive)
about, v1.5.2 (active)
archive-view, v0.61.1 (active)
autocomplete-atom-api, v0.10.0 (active)
autocomplete-css, v0.11.1 (active)
autocomplete-html, v0.7.2 (active)
autocomplete-plus, v2.31.0 (active)
autocomplete-snippets, v1.11.0 (active)
autoflow, v0.27.0 (inactive)
autosave, v0.23.1 (active)
background-tips, v0.26.0 (active)
bookmarks, v0.41.0 (active)
bracket-matcher, v0.82.1 (active)
command-palette, v0.38.0 (inactive)
deprecation-cop, v0.54.1 (active)
dev-live-reload, v0.47.0 (active)
encoding-selector, v0.22.0 (active)
exception-reporting, v0.39.0 (active)
fuzzy-finder, v1.3.0 (active)
git-diff, v1.1.0 (active)
find-and-replace, v0.198.0 (inactive)
go-to-line, v0.31.0 (inactive)
grammar-selector, v0.48.1 (active)
image-view, v0.58.0 (active)
incompatible-packages, v0.26.1 (active)
keybinding-resolver, v0.35.0 (active)
line-ending-selector, v0.5.0 (active)
link, v0.31.1 (inactive)
markdown-preview, v0.158.0 (active)
metrics, v0.53.1 (active)
notifications, v0.64.1 (active)
open-on-github, v1.2.0 (inactive)
package-generator, v1.0.0 (inactive)
settings-view, v0.238.0 (active)
snippets, v1.0.2 (active)
spell-check, v0.67.1 (active)
status-bar, v1.4.1 (active)
styleguide, v0.46.0 (active)
symbols-view, v0.113.0 (inactive)
tabs, v0.98.0 (active)
timecop, v0.33.1 (active)
tree-view, v0.208.0 (active)
update-package-dependencies, v0.10.0 (active)
welcome, v0.34.0 (active)
whitespace, v0.32.2 (active)
wrap-guide, v0.38.1 (active)
language-c, v0.52.1 (active)
language-clojure, v0.21.0 (active)
language-coffee-script, v0.47.0 (active)
language-csharp, v0.12.1 (active)
language-css, v0.36.2 (active)
language-gfm, v0.86.0 (active)
language-git, v0.13.0 (active)
language-go, v0.42.0 (active)
language-html, v0.44.1 (active)
language-hyperlink, v0.16.0 (active)
language-java, v0.22.0 (active)
language-javascript, v0.119.0 (active)
language-json, v0.18.0 (active)
language-less, v0.29.3 (active)
language-make, v0.22.2 (active)
language-mustache, v0.13.0 (active)
language-objective-c, v0.15.1 (active)
language-perl, v0.35.0 (active)
language-php, v0.37.0 (active)
language-property-list, v0.8.0 (active)
language-python, v0.45.0 (active)
language-ruby, v0.68.5 (active)
language-ruby-on-rails, v0.25.0 (active)
language-sass, v0.52.0 (active)
language-shellscript, v0.22.3 (active)
language-source, v0.9.0 (active)
language-sql, v0.21.1 (active)
language-text, v0.7.1 (active)
language-todo, v0.28.0 (active)
language-toml, v0.18.0 (active)
language-xml, v0.34.8 (active)
language-yaml, v0.26.0 (active)

# Dev
No dev packages
```

**Comment by sadovnychyi** [comment]
There's no way to ignore any files since it's managed by Jedi. I'm curious
if it's possible to reproduce it there, without involving
autocomplete-python. But that invalid token exception is definitely an
issue of this package.

**Comment by KyleKing** [comment]
Thanks for the quick reply. I actually disabled autocomplete_python and I no longer get the errors

**Comment by sadovnychyi** [comment]
Should be fixed in 1.8.9.
